README.md

```diff
@@ -1,7 +1,252 @@
+partial) `postgresql://` database url. You can leave out username and
+database name as these get overwritten from `config/database.yml`
+anyway.
+
+Once your environment variables are set up, you can run
+
+```sh
+RAILS_ENV=production bin/rails db:setup
+```
+
+#### Step 5: Precompile Assets
+
+Run the followin command to precompile CSS and JavaScript files:
+
+```sh
+RAILS_ENV=production bin/rails assets:precompile
+```
+
+#### Step 6: Start fediscoverer on System (Re)Start
+
+You need to run two processes, the fediscoverer web application server
+(puma) and the background job runner (Solid Queue).
+
+We recommend using your operating system's process supervisor to do so
+and make sure those processes are started on system startup.
+
+You can find example files for systemd in the [`examples/`](examples/)
+directory. Make sure to add / replace the environment variables in there
+with the correct values for your system.
+
+#### Step 7: Configure a Frontend Webserver
+
+We recommend you run the fediscoverer web application server behind a
+full-fledged web server that acts as a reverse proxy, handles SSL/TLS
+and serving of static files.
+
+A popular choice of web server for Ruby on Rails applications is nginx,
+but any webserver with the aforementioned features should do.
+
+### Docker
+
+The provided [`Dockerfile`](Dockerfile) should be sufficient to run both
+the ruby application server (started by default) and the background job
+runner (in which you need to run `bin/jobs` instead of `bin/rails
+server`).
+
+This allows you to skip some of the steps of a manual installation, but
+you still need most of them, so please read them carefully.
+
+We also push ready-made container images to Github's registry.
+Please see [packages](https://github.com/mastodon/fediscoverer/pkgs/container/fediscoverer).
+
+### Kubernetes
+
+We will provide an official Helm chart to run fediscoverer. This is
+coming soon. Watch this space.
+
+## Configuration
+
+fediscoverer can be configured using environment variables. Here is a
+(possibly incomplete) list:
+
+| Variable | Mandatory | Default | Description |
+| -------- | --------- | ------- | ----------- |
+| `SECRET_KEY_BASE` | Yes |  | Random key, unique per installation |
+| `DOMAIN` | Yes | | The domain name under which the web application is accessible |
+| `FEDISCOVERER_DATABASE_PASSWORD` | No | | Password of the `fediscoverer` user used to access the databases |
+| `DATABASE_URL` | No |  | Optional PostgreSQL URL to set database host, port etc. |
+| `WEB_CONCURRENCY` | No | `1` | Number of puma application server processes. Can be set to the number of CPUs to make use of all of them |
+| `JOB_CONCURRENCY` | No | `1` | Number of Solid Queue processes. Can be set to the number of CPUs to make use of all of them |
+| `MISSION_CONTROL_USERNAME` | No | | HTTP basic authentication username to access the [Mission Control - Jobs](https://github.com/rails/mission_control-jobs) web interface |
```

Comment:
If not configured, is mission control accessible by all?

Reply:
No.